src/utils.ts

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+
+export enum DomainType {
+  APP_DOMAIN,
+  LEGACY_DOMAIN,
+  NON_HUB_DOMAIN,
+}
+
+export const detectAppDomainType = (): DomainType => {
+  const { hostname: hn } = window.location;
+
+  if (hn.includes('projectdabl') && hn.includes('.com')) {
+    console.log('LEGACY DOMAIN DETECTED');
```

Comment:
Dunno if these `console.log` statements should remain…

Reply:
yeah, good point. 

as a whole, I've been wondering how to handle logging in the library. I don't really know how other libraries usually handle this sort of thing, but it might make sense to make logging output configurable in some way.

As a lib user, I might not want my prod app builds to include the lib's default logging onto the browser console, but I might want to enable them for my dev builds. Do you know how this kind of thing is usually handled? Maybe via an environment variable that is read during build time?